client/me/security-checkup/account-email.jsx

```diff
@@ -7,12 +7,14 @@ import {
 	isCurrentUserEmailVerified,
 } from 'calypso/state/current-user/selectors';
 import getUserSettings from 'calypso/state/selectors/get-user-settings';
+import hasUserSettings from 'calypso/state/selectors/has-user-settings';
 import isPendingEmailChange from 'calypso/state/selectors/is-pending-email-change';
 import { getOKIcon, getWarningIcon } from './icons.js';
 import SecurityCheckupNavigationItem from './navigation-item';
```

Comment:
I know it is not your code, but does it worth to do not use relative paths? (same comment applies to the other files)

```suggestion
import { getOKIcon, getWarningIcon } from 'calypso/me/security-checkup/icons.js';
import SecurityCheckupNavigationItem from 'calypso/me/security-checkup/navigation-item';
```

Reply:
Probably. I'd rather all of the files in the directory were refactored into TypeScript, but I wanted to make sure this is working and shipped first! 🙃 
So that's likely to be an exercise for another week.